lib/trino-parquet/src/main/java/io/trino/parquet/writer/valuewriter/DecimalValueWriter.java

```diff
@@ -73,14 +73,16 @@ else if (decimalType.isShort()) {
 
     private byte[] paddingBigInteger(BigInteger bigInteger)
     {
-        byte[] result = new byte[getTypeLength()];
-        if (bigInteger.signum() < 0) {
-            Arrays.fill(result, (byte) 0xFF);
-        }
+        int numBytes = getTypeLength();
```

Comment:
> Simplify logic of DecimalValueWriter#paddingBigInteger

The class is later removed, so maybe don't change it?

Reply:
We still use that logic in `BinaryLongDecimalValueWriter`